js/app.js

```diff
@@ -11,6 +11,26 @@ alert("You will answer with a yes or no");
 let points = 0;
 
 
+// let answer = prompt("Do I Like Sports");
+
+
+function sports (answer){
+
+if (answer.toLowerCase() === "yes") alert("Yes I like Sports!");
+else if (answer.toLowerCase() === "no") alert("Wrong Of Course I Like Sports!");
+// console.log(answer);
+}
+sports(answer);
```

Comment:
If you're making a new function for each question, consider running the prompt inside the function, then you don't need to have answer as an argument.